webdriver/tests/interop/beforeunload_prompt.py

```diff
@@ -59,59 +58,46 @@ def sync_navigate():
             opened_event,
         )
 
-        # Close the beforeunload prompt and wait for the navigation to finish.
-        await bidi_session.browsing_context.handle_user_prompt(
-            context=new_tab["context"], accept=accept
-        )
         closed_event = await wait_for_future_safe(on_prompt_closed)
-        await task_navigate
-
-        # Check that the beforeunload prompt is closed and the event was sent.
-        with pytest.raises(NoSuchAlertException):
-            current_session.alert.text
-
         recursive_compare(
             {
-                "accepted": accept,
+                "accepted": True,
                 "context": new_tab["context"],
                 "type": "beforeunload",
             },
             closed_event,
         )
 
-        if accept:
-            assert current_session.url == page_target
-        else:
-            assert current_session.url == page_beforeunload
+        # Assert the alert cannot be closed via classic.
```

Comment:
The comment doesn't match. Here we only check that the `beforeunload` prompt is not present anymore.

Reply:
fixed